inc/class-elementor.php

```diff
@@ -91,6 +91,7 @@ public function enqueue_editor_scripts() {
 
 		wp_enqueue_script( 'analogwp-elementor-modal', ANG_PLUGIN_URL . 'assets/js/elementor-modal.js', array( 'jquery' ), filemtime( ANG_PLUGIN_DIR . 'assets/js/elementor-modal.js' ), false );
 		wp_enqueue_style( 'analogwp-elementor-modal', ANG_PLUGIN_URL . 'assets/css/elementor-modal.css', array( 'dashicons' ), filemtime( ANG_PLUGIN_DIR . 'assets/css/elementor-modal.css' ) );
+		wp_enqueue_style( 'ang-sk-main', ANG_PLUGIN_URL . 'assets/css/sk-main.css', array(), filemtime( ANG_PLUGIN_DIR . 'assets/css/sk-main.css' ) );
```

Comment:
Instead of enqueuing this script twice, wouldn't it better to register once and enqueue wherever needed?

Reply:
Those are both two different hooks and registering them at `admin_enqueue_scripts` hook won't work at `elementor/editor/before_enqueue_scripts`. Will have to register twice evidently making it the same.